src/api/handlers/PetDeleteHandler.ts

```diff
@@ -0,0 +1,28 @@
+import { Handler } from "aws-lambda";
+import { PetService } from "../services/PetService";
+
+class PetDeleteHandler {
+    private service: PetService;
+
+    constructor() {
+        this.service = new PetService();
+    }
+
+    public execute(event: any) {
+        this.service.remove(event);
```

Comment:
@Aristevao just get the `id` from `event` object with: `event.pathParameters.id`